index.js

```diff
@@ -0,0 +1,99 @@
+const formToggler = document.getElementById('formToggler');
+const form = document.getElementById('form');
+const titleInput = document.getElementById('titleInput');
+const authorInput = document.getElementById('authorInput');
+const pagesInput = document.getElementById('pagesInput');
+const checkboxInput = document.getElementById('readCheckbox');
+const bookList = document.getElementById('bookList');
+
+let myBooks = [];
+
+function Book(title = '', author = '', pages = '', read = false) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
+}
+
+function createBookCard(book) {
+  const bookCard = document.createElement('div');
+  bookCard.setAttribute('data-title', book.title);
+
+  const bookTitle = document.createElement('h3');
+  const bookAuthor = document.createElement('h3');
+  const bookPages = document.createElement('p');
+  const readStatus = document.createElement('p');
+
+  const removeButton = document.createElement('button');
+  removeButton.setAttribute('type', 'button');
+  removeButton.innerText = 'Remove book';
+
+  const readButton = document.createElement('button');
+  readButton.setAttribute('type', 'button');
+  readButton.innerText = book.read ? 'Unread' : 'Read';
+
+  bookCard.classList.add('book-card');
+  bookTitle.innerText = `Title: ${book.title}`;
+  bookAuthor.innerText = `Author: ${book.author}`;
+  bookPages.innerText = `Pages: ${book.pages}`;
+  readStatus.innerText = `Read: ${book.read}`;
+
+  bookCard.append(bookTitle);
```

Comment:
- **[OPTIONAL]** To improve the performance and avoid multiple [reflows](https://developers.google.com/speed/docs/insights/browser-reflow?csw=1) every time we append an item, we can use [`createDocumentFragment`](https://developer.mozilla.org/en-US/docs/Web/API/Document/createDocumentFragment).